plugin/pkg/scheduler/api/types.go

```diff
@@ -70,7 +70,7 @@ type PriorityPolicy struct {
 	Argument *PriorityArgument
 }
 
-// Represents the arguments that the different types of predicates take
+// PredicateArgument represents the arguments that the different types of predicates take
```

Comment:
Now that you are at it, could you fix the comments as well:

PredicateArgument represents the arguments to configure predicate functions in scheduler policy configuration.